package.json

```diff
@@ -27,7 +27,11 @@
     "@babel/preset-flow": "^7.10.4",
     "@babel/register": "^7.10.5",
     "@babel/runtime": "^7.11.0",
-    "@rollup/plugin-json": "4.0.1",
+    "@rollup/plugin-babel": "^5.2.0",
+    "@rollup/plugin-commonjs": "^13.0.0",
```

Comment:
The latest `commonjs` is 15, however I have to pin to 13 due to an upstream issue: https://github.com/rollup/plugins/issues/557